cmd/rekor-server/app/root.go

```diff
@@ -98,9 +98,11 @@ func init() {
 	rootCmd.PersistentFlags().String("rekor_server.address", "127.0.0.1", "Address to bind to")
 
 	rootCmd.PersistentFlags().String("rekor_server.signer", "memory",
-		`Rekor signer to use. Valid options are: [awskms://keyname, azurekms://keyname, gcpkms://keyname, hashivault://keyname, memory, <filename containing PEM-encoded private key>].
+		`Rekor signer to use. Valid options are: [awskms://keyname, azurekms://keyname, gcpkms://keyname, hashivault://keyname, memory, tink, <filename containing PEM-encoded private key>].
 Memory and file-based signers should only be used for testing.`)
 	rootCmd.PersistentFlags().String("rekor_server.signer-passwd", "", "Password to decrypt signer private key")
+	rootCmd.PersistentFlags().String("rekor_server.tink_kek_uri", "", "Key encryption key for decrypting Tink keyset. Valid options are [aws-kms://keyname, gcp-kms://keyname]")
```

Comment:
could we use `gcpkms` to match above or is this hyphen hardcoded into the tink library?

Reply:
No unfortunately, these are the values hardcoded in Tink.

Reply:
https://github.com/tink-crypto/tink-go-gcpkms/blob/main/integration/gcpkms/gcp_kms_client.go#L33C2-L33C11